markdown/chapters/meandeck/basics.md

```diff
@@ -4,74 +4,138 @@ order: 0
 title: The Basics
 ---
 
-This section explains the basic Doomsday piles that attempt to end the game
-directly with a {{Thassa's Oracle}} trigger. While older versions of Meandeck
-used {{Laboratory Maniac}} as the primary win-condition, the introduction of TO
-allows us to build piles that are more streamlined than ever.
+The goal of all Magic decks is to win the game. {{Doomsday}} aims to do so by
+casting its namesake card and constructing a *pile* of 5 cards tailored to the
+current game state. While it would be nigh impossible to enumerate every game
+state and pile variation, these chapters are aimed to show you the basic
+concepts behind casting and winning with Doomsday. There is one primary win
+condition that is used in this version of the deck to win with and that is via
+{{Thassa's Oracle}}.
+
+## Pile Construction Steps
 
 A *pile* in this context is what we use to describe the stack of five cards that
-comprise your library after the resolution of Doomsday.
+comprise your library after the resolution of Doomsday. This is terminology that
+will be used throughout the Wiki.
 
-As a general rule, in order to win you need to:
+There are three steps that need to occur to enable you to win (at least most of
+the time).  
+These are as follows:
 
 1. Resolve Doomsday
-1. Draw into your pile
-1. Consume enough cards out of the pile so that TO's trigger wins you the game
-
-Often enough, steps 2 and 3 go together by using a *cantrip* effect. This is
-typically a draw spell like Preordain or a free cycle effect like Street Wraith,
-which helps *dig* into the pile. This can be done in a single turn or spread
-over multiple turns, utilising your draw step as a way to *dig* into the pile.
+1. Draw into your pile enough so that the Oracle's trigger can win you the game
+1. Resolve Thassa's Oracle
+
+<row variant="centered">{{!DD}}{{!TO}}</row>
+
+Out these steps, we will start by detailing how to achieve steps 2. and 3. as
```

Comment:
> Out these steps

Is that equivalent to "with these out of the way"? I wasn't sure whether that's an idiom or a typo 🤔

Reply:
Reworded, was tired, poorly written me's fault